src/loaders/ObjectLoader.js

```diff
@@ -817,6 +819,13 @@ ObjectLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 				break;
 
+			case 'LightProbe':
+
+				var sh = new SphericalHarmonics3().fromArray( data.sh );
+				object = new LightProbe( sh, data.intensity );
```

Comment:
fwiw I'm putting toJSON and fromJSON on LightProbeVolume: 

https://github.com/mrdoob/three.js/blob/c704ff0ebe1a42d7f97f2cf2c22f2d14be50de67/examples/jsm/lights/LightProbeVolume.js#L345-L410

Reply:
So you suggest to introduce `LightProbe.fromJSON()`, right?

Reply:
I'd suggest https://github.com/mrdoob/three.js/issues/11266 as a long-term goal. But I won't insist on this PR. :)

Reply:
Okay, I've aligned the PR to yours. I think it makes sense if both follow the same approach 👍